### PR TITLE
Backbone Metal

### PR DESCRIPTION
**Pull request by thejameskyle**
Opening this to give you guys an idea of what I'm working on. Tests are super duper broken

### CODE REVIEW

File: test/unit/view.ui-event-and-triggers.spec.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ describe('view ui event trigger configuration', function() {
...
       beforeEach(function() {
         this.View = Marionette.ItemView.extend({
-          template : this.templateFn,
-          ui       : this.sinon.stub().returns(this.uiHash),
-          triggers : this.sinon.stub().returns(this.triggersHash),
-          events   : this.sinon.stub().returns(this.eventsHash)
+          template: this.templateFn,
+          ui: this.sinon.stub().returns(this.uiHash),
+          triggers: this.sinon.stub().returns(this.triggersHash),
+          events: this.sinon.stub().returns(this.eventsHash)
```
Comment (thejameskyle) [inline test/unit/view.ui-event-and-triggers.spec.js]: goddamnit atom.

Comment (samccone) [inline test/unit/view.ui-event-and-triggers.spec.js]: bahahahah

Comment (samccone) [inline test/unit/view.ui-event-and-triggers.spec.js]: but it is the hackable editor ©

**Comment by samccone** [comment]
rebase plz @thejameskyle 

aka just cherry-pick

**Pull request closed without merging**